lua/starfall/editor/sfframe.lua

```diff
@@ -2045,13 +2045,7 @@ function PANEL:UpdatePlayers(players)
 			local killserver = vgui.Create("StarfallButton", cpuServer)
 			killserver:SetText("Admin Kill")
 			killserver.DoClick = function()
-				if SF.playerInstances[ply] then
-					for instance, _ in pairs(SF.playerInstances[ply]) do
-						net.Start("starfall_processor_kill")
-						net.WriteEntity(instance.entity)
-						net.SendToServer()
-					end
-				end
+				RunConsoleCommand( "sf_user_terminate", steamid )
```

Comment:
Could we make this command simpler? sf_kill and sf_kill_cl sounds fine.

Reply:
Yeah, the `user` part is because of how the block commands are structured in the `_call` for `sflib`